Documentation/En/Index.xyl

```diff
@@ -31,9 +31,10 @@
     the SUT after its execution the ones we expect or not?</li>
   </ol>
   <p>The values of the verdict are: <strong>Success</strong>,
-  <strong>fail</strong> or <strong>inconclusive</strong>. A success means this
-  is what we expect, a fail means this is not what we expect and an
-  inconclusive means this is not possible to give an answer. The important
+  <strong>fail</strong> or <strong>inconclusive</strong>. "Success" means that
```

Comment:
Please, use `“Success”` (real English quotes) instead of `"Success"` (straight coding quotes).